packages/jest-environment/package.json

```diff
@@ -49,11 +49,11 @@
 		"testEnvironment": "./lib"
 	},
 	"dependencies": {
-		"@jest/environment": "^27.5.1",
-		"@jest/fake-timers": "^27.5.1",
-		"@jest/types": "^27.5.1",
-		"jest-mock": "^27.5.1",
-		"jest-util": "^27.5.1",
+		"@jest/environment": "^29.4.0",
+		"@jest/fake-timers": "^29.4.0",
+		"@jest/types": "^29.4.0",
```

Comment:
this shouldn't be a dependency, it needs to be a devDependency so that it's not installed for end users of the package.

Reply:
same goes for `@jest/environment`?

Reply:
@jereklas I think `@jest/environment` has to be a dependency here. `@jest/types` can be a dev dependency.